static/sass/styles.scss

```diff
@@ -232,3 +230,13 @@ $p-small-lh-diff: map-get($line-heights, default-text) -
     }
   }
 }
+
+// Style drowdown sub-links to match the other links in the navigation
+ul.p-navigation__dropdown-sub-list {
```

Comment:
@petesfrench I think it's worth opening an issue for this in Vanilla, I'm surprised it doesn't already have something for this use case (or maybe it does, but it's hidden away and needs to be documented).